exercises/error-handling/error_handling_test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+import error_handling as er
+
```

Comment:
Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if aplicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Reply:
At this time, there is no canonical data for this exercise; these tests were adapted from the [C# implemetation of this exercise](https://github.com/exercism/csharp/blob/f838767eab422738c8d8517b910dfcd926602969/exercises/error-handling/ErrorHandlingTest.cs). Is there a standard way of noting this?

Reply:
If there is no tests in problem-specifications repo for this exercise, then it's Ok to leave it as is.

We agreed with @N-Parsons to use comment above to denote the version of `canonical-data.json` from which tests were adopted